doc/source/configuration/magnum-capi.rst

```diff
@@ -1,45 +1,127 @@
+
+- Install k3s on the seed (with all k3s data stored on the attached Cinder volume)
+
+- Install the required components on the k3s cluster to provision the HA cluster
+
+- Provision the HA cluster
+
+- Install the required components on the HA cluster to manage Magnum user clusters
+
+Once the seed VM has been provisioned, it can be accessed via SSH by running ``./bin/seed-ssh`` from the root of the azimuth-config repository. Within the seed VM, the k3s cluster and the HA cluster can both be accessed using the pre-installed ``kubectl`` and ``helm`` command line tools. Both of these tools will target the k3s cluster by default; however the ``kubeconfig`` file for the HA cluster can be found in the seed's home directory (named e.g. ``kubeconfig-capi-mgmt-<site-specific-name>.yaml``).
+
+*Note* - The provision playbook is responsible for copying the HA ``kubeconfig`` to this location *after* the HA cluster is up and running. If you need to access the HA cluster while it is still deploying, the ``kubeconfig`` file can be found stored as a Kubernetes secret on the k3s cluster.
+
+It is possible to reconfigure or upgrade the management cluster after initial deployment by simply re-running the ``provision_capi_mgmt`` playbook. However, it's preferable that most Day 2 ops (i.e. reconfigures and upgrades) be done via a CD Pipeline. See `these Azimuth docs <https://stackhpc.github.io/azimuth-config/deployment/automation/>`_ for more information.
 
-It's preferable that most Day 2 ops be done via a `CD Pipeline <https://stackhpc.github.io/azimuth-config/deployment/automation/>`__.
 
 Kayobe Config
 ==============
-Ensure that your kayobe-config branch is up to date on |current_release_git_branch_name|.
 
-Copy the kubeconfig found at `kubeconfig-capi-mgmt-<your-az-environment>.yaml` to your kayobe environment (e.g. `<your-skc-environment>/kolla/config/magnum/kubeconfig`. It is highly likely you'll want to add this file to ansible vault.
+To configure the Magnum service with the Cluster API driver enabled, first ensure that your kayobe-config branch is up to date with |current_release_git_branch_name|.
 
-Ensure that your magnum.conf has the following set:
+Next, copy the CAPI management cluster's kubeconfig file into to your stackhpc-kayobe-config environment (e.g. ``<your-skc-environment>/kolla/config/magnum/kubeconfig``). This file must be Ansible vault encrypted.
 
-.. code-block:: yaml
+The following config should also be set in your stackhpc-kayobe-config environment:
+
+.. code-block:: ini
+    :caption: magnum.conf
 
     [nova_client]
     endpoint_type = publicURL
 
+.. code-block:: yaml
+    :caption: kolla/globals.yml
 
-This is used to generate the application credential config injected into the tenant Kubernetes clusters, such that it is usable from within an OpenStack project, so you can't use the "internal API" end point here.
+    magnum_cluster_api_driver_enabled: true
```

Comment:
Kolla-ansible sets this when kubeconfig exists.

Reply:
Only in `stackhpc/kolla-ansible/stackhpc/yoga` apparently. Will leave this in for now and update in future once Kolla-Ansible upstream has the required functionality to set this automatically.